analysed-packages/jquery-mousewheel/version-3.1.13/jquery-mousewheel-3.1.13-SPDX2TV.spdx

```diff
@@ -0,0 +1,389 @@
+SPDXVersion: SPDX-2.3
+DataLicense: CC0-1.0
+
+##-------------------------
+## Document Information
+##-------------------------
+
+DocumentNamespace: http://fossology-6674c79fbf-2qzn2/repo/SPDX2TV_jquery-mousewheel-67289b6b2aa0066d7d78a5807f520387135ffb22.zip_1713540928.spdx
```

Comment:
This is a strange name. It seems to be a certain commit but not the released version 3.1.13 as mentioned in the download loaction in the README.md file. I downloaded the package and the directory names are:
jquery-mousewheel-3.1.13.zip/jquery-mousewheel-3.1.13/test/browserify for example.
This comment is valid for all file paths

Reply:
The hash refers to that exact version. https://github.com/jquery/jquery-mousewheel/releases/tag/3.1.13
has this commit 67289b6b2aa0066d7d78a5807f520387135ffb22. It was uploaded with the wrong name but the content are the same. I have changed it and thank you!